src/pattern-library/components/patterns/prototype/SelectNextPage.tsx

```diff
@@ -450,6 +450,21 @@ export default function SelectNextPage() {
               withSource
             />
           </Library.Example>
+          <Library.Example title="onListboxScroll">
+            <Library.Info>
+              <Library.InfoItem label="description">
+                A callback passed to the listbox <code>onScroll</code>.
+              </Library.InfoItem>
+              <Library.InfoItem label="type">
+                <code>
+                  () {'=>'} void {'|'} undefined
```

Comment:
This callback actually receives an `Event`, although I'm not sure whether you want to promise that in the API.